lumibot/components/agents/builtins.py

```
@@ -220,6 +263,126 @@ def _require_agent_order_readiness(symbol: str) -> None:
         )
 
 
+def _parse_symbol_list(
```

Pylint (4.0.7)

[refactor] Too many branches (16/12)

(R0912)